src/Foliage-Core/FOBlogPost.class.st

```diff
@@ -4,17 +4,66 @@ Class {
 	#category : #'Foliage-Core-Blog'
 }
 
-{ #category : #'as yet unclassified' }
+{ #category : #accessing }
+FOBlogPost >> abstract [
+
+	^ super abstract
+]
+
+{ #category : #accessing }
+FOBlogPost >> abstractLayout [
```

Comment:
I think I need to work here a bit. The structure of the document should usable and every part can have its own layout